src/PostalRegistry.Api.Legacy/PostalInformation/Query/PostalInformationLinkedDataEventStreamQuery.cs

```diff
@@ -7,20 +7,18 @@ namespace PostalRegistry.Api.Legacy.PostalInformation.Query
     using NodaTime;
     using PostalRegistry.Projections.Legacy;
     using PostalRegistry.Projections.Legacy.PostalInformationLinkedDataEventStream;
-    using PostalRegistry.Projections.Legacy.PostalInformationSyndication;
     using System;
     using System.Collections.Generic;
     using System.Linq;
     using System.Linq.Expressions;
-    using System.Threading.Tasks;
 
     public class PostalInformationLinkedDataEventStreamQueryResult
     {
         public string PostalCode { get; }
         public string ChangeType { get; }
         public Instant EventGeneratedAtTime { get; }
         public PostalInformationStatus? Status { get; }
-        public IEnumerable<PostalName>? PostalNames { get; }
```

Comment:
Why was this nullable?

Reply:
Because it is possible that the column PostalNames is NULL.

Reply:
DB can be null, but you EF mapping doesn't return null.
https://github.com/Informatievlaanderen/postal-registry/pull/177/files#diff-fd5d4928db3636d9695a4cc1554c28416631464989a073747c59527efdd73787R77-R80